src/main/java/server/response/ResponseFormatter.java

```diff
@@ -0,0 +1,44 @@
+package server.response;
+
+import java.util.ArrayList;
+
+public class ResponseFormatter {
+  private static final String crlf = "\r\n";
```

Comment:
Let's follow the [Java convention for constants](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)